descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "inventory",
-      "version": "11.5",
+      "version": "12.0",
```

Comment:
This change should have been done together with a major implementation version change (to 19.0.0 I believe). 

Please raise a follow up pull request to do so.